dev-client/src/components/modals/SignOutModal.tsx

```diff
@@ -26,20 +26,20 @@ import {useDispatch} from 'terraso-mobile-client/store';
 
 export type LogoutModalProps = Pick<ModalProps, 'trigger'>;
 
-export function LogoutModal({trigger}: LogoutModalProps) {
+export function SignOutModal({trigger}: LogoutModalProps) {
```

Comment:
Change suggested by @paulschreiber on previous PR, I was passing by the settings screen again so I decided to just do it.